src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -0,0 +1,200 @@
+              {
+                actorEmail: <strong>{actor.email}</strong>,
+                accountSettings: <Link to="/settings/account/emails/" />,
+              }
+            )}
+          </EmailAlert>
+        )}
+      </React.Fragment>
+    }
+    body={
+      <HovercardBody>
+        {commits !== undefined && (
+          <React.Fragment>
+            <div className="divider">
+              <h6>{t('Commits')}</h6>
+            </div>
+            <ReasonGroup>
+              {commits.slice(0, 6).map(({message, dateCreated}, i) => (
+                <ReasonItem key={i}>
+                  <CommitIcon />
+                  <CommitMessage message={message} date={dateCreated} />
+                </ReasonItem>
+              ))}
+            </ReasonGroup>
+          </React.Fragment>
+        )}
+        {rules !== undefined && (
+          <React.Fragment>
+            <div className="divider">
+              <h6>{t('Ownership Rules')}</h6>
+            </div>
+            <ReasonGroup>
+              {rules.map(([type, matched], i) => (
+                <ReasonItem key={i}>
+                  <OwnershipTag tagType={type} />
+                  <OwnershipValue>{matched}</OwnershipValue>
+                </ReasonItem>
+              ))}
+            </ReasonGroup>
+          </React.Fragment>
+        )}
+      </HovercardBody>
+    }
+    {...props}
+  />
+);
+
+SuggestedOwnerHovercard.propTypes = {
+  /**
+   * The suggested actor.
+   */
+  actor: PropTypes.oneOfType([
+    // eslint-disable-next-line react/no-typos
+    SentryTypes.User,
+    // Sentry user which has *not* been expanded
+    PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+    // Unidentifier user (from commits)
+    PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+    // Sentry team
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+  ]),
+
+  /**
+   * The list of commits the actor is suggested for. May be left blank if the
+   * actor is not suggested for commits.
+   */
+  commits: PropTypes.arrayOf(
+    PropTypes.shape({
+      message: PropTypes.string.isRequired,
+      dateCreated: PropTypes.string.isRequired,
+    })
+  ),
+
+  /**
+   * The list of ownership rules the actor is suggested for. Maybe left blank
+   * if the actor is not suggested based on ownership rules.
+   */
+  rules: PropTypes.arrayOf(PropTypes.array),
+};
+
+const tagColors = {
+  url: 'greenLight',
+  path: 'blueLight',
+};
+
+const CommitIcon = styled(p => <InlineSvg src="icon-commit" size="16px" {...p} />)`
+  margin-right: 4px;
+  flex-shrink: 0;
+`;
+
+const CommitMessage = styled(({message, date, ...props}) => (
+  <div {...props}>
```

Comment:
you might get some warnings here unless you use `shouldFordwardProps`?

Reply:
It should be fine since I destructed the props it needs.